frontend/alanda/libs/common/src/lib/permissions/permissions.directive.ts

```diff
@@ -29,20 +26,21 @@ interface AlandaPermissionsDirectiveState {
   providers: [RxState],
 })
 export class AlandaPermissionsDirective {
+
   hostElementManagers: ElementManager[] = getManagersByElementRef(
     this.hostElement,
   );
 
   @Input('alandaPermissions')
   set permission(permissionString: string) {
-    this.rxState.set({ permissionString });
+    this.rxState.set({permissionString});
```

Comment:
This is not properly formatted. Can you run `npm run format:write` to format your code.